main/src/ca/uwaterloo/flix/api/lsp/provider/CompleteProvider.scala

```diff
@@ -37,8 +66,22 @@ object CompleteProvider {
+    CompletionItem("elif", "else if (${1:/* condition */}) {\n    ${2:/* code */}\n}", None, Some("code snippet for else if ()"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("else", "else {\n    ${1:/* code */}\n}", None, Some("code snippet for else"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("enum", "enum ${1:Sample} {\n    case ${2:DataType1}\n    ${4:/* add more algebric data types separated by comma (,) */}\n}", None, Some("code snippet for enum"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("if", "if (${1:/* condition */}) {\n    ${2:/* code */}\n}", None, Some("code snippet for if ()"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("import", "import ${1:module}", None, Some("preprocessor keyword"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("instance", "instance ${1:Sample}[Option[${2:a}]] with $1[$2] {\n    ${3:/* code */}\n}", None, Some("code snippet for instance"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("main", "def main(args: Array[String]) : Int32 & Impure = \n    ${1:/* code */} \n    0", None, Some("code snippet for main ()"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("main Hello World", "def main(_args: Array[String]) : Int32 & Impure = \n    println(\"Hello World!\");\n    0", None, Some("code snippet for Hello World Program"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("match", "match ${1:exp} {\n    case ${2:pat} => ${3:exp}\n}", None, Some("code snippet for pattern match"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("namespace", "namespace ${1:sample} {\n    ${2:/* code */} \n}", None, Some("code snippet to create namespace"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("opaque", "opaque type ${1:name} = ${2:type}", None, Some("code snippet for opaque type"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("println", "println($1);", None, Some("print to standard output"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, List("\"")),
     CompletionItem("query", "query ${1:db} select ${2:cols} from ${3:preds} ${4:where ${5:cond}}", None, None, CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("rel", "rel ${1:Sample}(${2:x}: ${3:type}, ${4:y}: ${5:type})", None, Some("code snippet to declare predicate symbol"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("type alias", "type alias ${1:name} = ${2:type}", None, Some("code snippet for type alias"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil)
   )
```

Comment:
I will give feedback on the snippets in the next iteration, but meanwhile I think a few more snippets could be added:

- class
- import could be split into multiple types (there is import new, import get, and so forth. Please explore that! :))
- let?
- and maybe let with type (e.g. let name: type = exp; \n exp).
- law? (see Order.flix)
- select (see flix.dev for example)